lib/packages/shared-utils/date-helper.test.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, it, expect, vi } from "vitest";
+import { formatDate, formatNinetyDaysDate, isDST } from "./date-helper";
+import { beforeEach } from "node:test";
+
+describe("date-helper", () => {
+  describe("formatDate", () => {
+    it('should return "Pending" if date is null or undefined', () => {
+      expect(formatDate(null)).toBe("Pending");
+      expect(formatDate(undefined)).toBe("Pending");
+    });
+
+    it("should correctly format a valid date", () => {
+      const date = new Date(2025, 0, 5); // Jan 5, 2025
+
+      expect(formatDate(date.getTime())).toBe("January 5, 2025");
+    });
+  });
+
+  describe("formatNinetyDaysDate", () => {
+    beforeEach(() => {
+      vi.spyOn(Date.prototype, "getTimezoneOffset").mockImplementation(function (this: Date) {
```

Comment:
We shouldnt mock out this function. We have it executed on the snapshots to use the one native to it.

Reply:
If you look at the vitest config for snapshot we do this

```typescript
  vi.useFakeTimers();
  const now = new Date(2023, 0, 1);
  vi.setSystemTime(now);
```

Reply:
It sets the mock date for today then when it hits the actual function itself it works as expected.

Reply:
Resolved with https://github.com/Enterprise-CMCS/macpro-mako/pull/1136/commits/9f8d05a18ac0c8306707278498339a7009c082eb